release/dist/synology/pkgs.go

```diff
@@ -42,9 +43,31 @@ func (t *target) Build(b *dist.Build) ([]string, error) {
 	return t.buildSPK(b, inner)
 }
 
+// dsmVersion combines major and minor version info into an int
+// representation.
+//
+// Version 7.2 becomes 72 as an example.
+func (t *target) dsmVersion() int {
+	return t.dsmMajorVersion*10 + t.dsmMinorVersion
+}
+
+// dsmVersionString returns a string representation of the version
+// including minor version information if it exists.
+//
+// If dsmMinorVersion is 0 this returns dsmMajorVersion as a string,
+// otherwise it returns "dsmMajorVersion-dsmMinorVersion".
+func (t *target) dsmVersionString() string {
+	dsmVersionString := fmt.Sprintf("%d", t.dsmMajorVersion)
+	if t.dsmMinorVersion != 0 {
+		dsmVersionString += fmt.Sprintf("-%d", t.dsmMinorVersion)
```

Comment:
Nit: if you don't want to combine string concatenation and printf in one statement, you could do this:

```
dsmVersionString = fmt.Sprintf("%s-%d", dsmVersionString, t.dsmMinorVersion)
```